docs/source/array-examples.rst

```diff
@@ -0,0 +1,9 @@
+Examples
+========
+
+.. toctree::
+   :maxdepth: 1
+
+   examples/array-random.rst
+   examples/array-numpy.rst
+   examples/array-pickle.rst
```

Comment:
I often go into docs looking for short examples I can experiment with to grok the concepts of a library, or to use as a starting point to build from. I find it convenient when they are centralized and easily accessible. Sometimes notebooks and blog posts go stale. These examples could also be tested as doctests if desired.